phpBB/styles/prosilver/theme/colours.css

```diff
@@ -409,14 +409,9 @@ dl.faq dt {
 /* Quote block */
 blockquote {
 	background-color: #EBEADD;
-	background-image: url("./images/quote.gif");
 	border-color:#DBDBCE;
 }
 
-.rtl blockquote {
-	background-image: url("./images/quote_rtl.gif");
-}
-
```

Comment:
This is also a breaking change if you remove the gifs so keep them for BC until 3.3